src/components/Atoms/Svg/__snapshots__/Icons.test.js.snap

```diff
@@ -1,5 +1,19 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
+exports[`<Icon /> Does not render an icon if an invalid name is provided 1`] = `
+<svg
```

Comment:
Should an icon with an invalid name render anything? Perhaps we should adjust the rendering logic in the Icon component so that, when the `iconPath` function returns null, we just return null (no svg element).